doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -346,6 +346,13 @@ nRF9160 samples
 Thread samples
 --------------
 
+* Changed:
+  * Overlay structure changed:
```

Comment:
```suggestion
* Changed:

  * Overlay structure changed:

```
This is likely what is breaking the documentation build, but the error is pointing to the wrong line.